### ISSUE TITLE
metadata for export table

### ISSUE THREAD

**Issue by cmahony** [bcgov/CCISS_ShinyApp#77]
the tool doesn't seem to provide metadata for the the export table. I made a [simple one](https://github.com/bcgov/CCISS_ShinyApp/blob/main/cciss_export_metadata.csv.xlsx) in commit 7412e9c7af3200aa0012d1da0bdad71fb9b3fc95. @kdaust please modify as needed and add to the tool. non-urgent.

**Comment by cmahony** [comment]
@kdaust looks like this is not resolved. could the metadata table be added to the csv download zip file? 
@destanycrane is there other information that should be downloaded with the data as a README (license, terms of use, etc.)?

**Comment by destanycrane** [comment]
@cmahony yes, a readme file should live with the data that includes links to the license (https://www2.gov.bc.ca/gov/content/data/policy-standards/open-data/open-government-licence-bc) and the Data Catalogue entry for metadata (once it's been published)

**Comment by kdaust** [comment]
This is fixed now. @destanycrane could you please edit the README as necessary? https://github.com/bcgov/CCISS_ShinyApp/blob/main/app/README.txt

**Comment by destanycrane** [comment]
@kdaust done! thank you
